jax/_src/scipy/special.py

```diff
@@ -956,3 +957,61 @@ def lpmn(m, n, z):
   p_derivatives = _gen_derivatives(p_vals, z, is_normalized)
 
   return (p_vals, p_derivatives)
+
+
+@partial(jit, static_argnums=(0, 1, 3))
```

Comment:
Same here: probably better not to add the JIT annotation at this level.

Reply:
Thanks! Done.